app/controllers/errors_controller.rb

```diff
@@ -4,7 +4,7 @@ class ErrorsController < ApplicationController
     define_method error_code.to_sym do
       respond_to do |format|
         format.any(:html, :text, :pdf, :mobi, :epub, :azw3) { render error_code, status: error_code.to_i }
```

Comment:
Looking at the existing code, the intent seems to be:

- for known formats (html, pdf, etc.) return the 404 page
- for all else return an empty response with just the status code

But it doesn't work out like that. `format.any(:html, :text, :pdf, :mobi, :epub, :azw3)` implies we have pdf versions of the error templates, which we don't.

I think you can replace the whole `respond_to` block with just:
```rb
render error_code, status: error_code.to_i, formats: :html
```
and it should work for the .htm case as well.

Reply:
Thanks a lot for the review. It looks like the header-only response for unknown formats was indeed part of the original intent, as seen in these changes in f03dbb8aa63bdb7ce8f3269fed7ee6bf518c8d47:
```diff
-        format.any(:html, :text, :pdf, :mobi, :epub) { render error_code, :status => error_code }
-        format.any(:js) { head error_code }
+        format.any(:html, :text, :pdf, :mobi, :epub) { render error_code, :status => error_code.to_i }
+        format.all { render :nothing => true, :status => error_code.to_i }
```

Just to make sure, are we going to drop this part as well

> for all else return an empty response with just the status code

and make it always render an HTML (as long as there is a template file for the given error code)?

Reply:
Yeah, I think it's friendlier to the user to render HTML if all else fails.

Reply:
Understood, thanks! The next commit should address this.